media-libs/chromaprint/chromaprint-1.4.3.recipe

```diff
@@ -52,6 +57,7 @@ BUILD()
 	cmake -DCMAKE_INSTALL_PREFIX=$prefix \
 		-DCMAKE_INSTALL_LIBDIR=$libDir \
 		-DCMAKE_INSTALL_INCLUDEDIR=$includeDir \
+		-DBUILD_TOOLS=ON \
```

Comment:
Check if it can use -DCMAKE_INSTALL_BINDIR also and maybe move the binary to a seperate _tools package?

Reply:
I added `-DCMAKE_INSTALL_BINDIR`, that is supported.

But I don't think a single 35K binary warrants the trouble for having a separate package. I would really prefer to keep this together.

Reply:
-DCMAKE_INSTALL_BINDIR isn't picked up by the project (gcc2h build), you probably would have to add a line in the CmakeList.txt file or use cmd:fpcalc without secondaryArchSuffix (as it's installed in $prefix/bin now)

Reply:
I only partially get what you mean here :) But I see how `-DCMAKE_INSTALL_BINDIR` gets ignored, I think it is even the same with `-DCMAKE_INSTALL_LIBDIR`. But the installed package seems to work, at least I can run `fpcalc` and it ends up in `/boot/system/bin/fpcalc`, and inside the package it is in `./bin/fpcalc` . Where should it be placed instead?

Reply:
If you build this for secondary architecture it "should" end up in $binDir
On gcc2h that would be /boot/system/bin/x86, but as you said it's now in /boot/system/bin (on primary x86_64 that is ok and you don't need the secondaryArchSuffix
PS for the lib and headers it's ok

Reply:
Created a PR to set this right :)